spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -53,27 +52,18 @@ def __init_spot(self):
             print("Failed to get lease for Spot. Exiting!")
             exit(1)
 
+        # TODO: Do we want to have a several yaml for nav/pick/place?
```

Comment:
@jimmytyyang  once we have merged the HardwareCI for nav,pick&place, creation of config will be much more cleaner.

Eg : https://github.com/facebookresearch/spot-sim2real/pull/65/files#diff-9e12c15ba22f9462f67b84c508262eb0ffb8fbd1852a5c8c193e514907934406R48-R57 We just use this method.

Reply:
Nice! roger that!

Reply:
@KavitShah1998  In addition, once we have the `controller` that wraps both `policy` and `env`, then we directly use `controller` in the skill manager